localstack/utils/bootstrap.py

```diff
@@ -360,6 +361,56 @@ def start_infra_locally():
     return infra.start_infra()
 
 
+def validate_localstack_config(name):
+    dirname = os.getcwd()
+    compose_file_name = '%s/%s' % (dirname, name)
```

Comment:
Let's better use `os.path.join`, for better compatibility. Also - ideally we should check whether `name` refers to an absolute file path
```
compose_file_name = name if os.path.isabs(name) else os.path.join(dirname, name)
```